src/backend/common/datafeed_parsers/json/json_team_list_parser.py

```diff
@@ -0,0 +1,24 @@
+from typing import List, Union
+
+from pyre_extensions import safe_json
+
+from backend.common.models.keys import TeamKey
+from backend.common.models.team import Team
+
+
+# Parsers probably need to live in common, since they'll also be used on the API side
```

Comment:
@ZachOrr we should agree on where these live, since there's some base code for parsers in `tasks-io` currently, but we'll also need the JSON parsers to be exposed over the trusted API

Reply:
I'm good moving them to common! They were in common to begin with, I moved them to tasks-io to see if we could keep them out of common. If we need to move them back, let's do it